projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loan/patron-profile-loan.component.html

```diff
@@ -45,7 +45,7 @@
         </div>
         <div *ngIf="record.metadata.extension_count && record.metadata.extension_count > 0" class="badge badge-secondary font-weight-normal">
           {{ record.metadata.extension_count }}
-          {{ record.metadata.extension_count | i18nPlural: {'=1': 'renewal', 'other': 'renewals'} | translate }}
+          {{ record.metadata.extension_count | i18nPlural: {'=1': 'Renewal', 'other': 'Renewals'} | translate | lowercase }}
```

Comment:
Why use First uppercase character and use `lowercase` filter next ?
It the reason is "because we already have translation for `Renewal`", it's a bad reason for me :-)